radar/infrastructure_ci_cd/styra_das.yaml

```diff
@@ -0,0 +1,29 @@
+name: Spinnaker
+blip:
+  - date: 2020-06-18
+    ring: ASSESS
+description: |
+  [Styra DAS](https://www.styra.com/manage-open-policy-agent-with-styra-das) is a SaaS management and
+  control plane for Open Policy Agent (OPA).
+rationale: |
+  Styra's Declarative Authorization Service (DAS) is a system to manage our OPA enabled services. It can load
+  service policies from GitHub, supports stack and libraries for policy reuse and differentiates between
+  ingress (Envoy integration) and application (service integration) policy declarations.
+
+  We're looking at Styra DAS because it acts both as a control plane and as a data bundle server for our
+  OPA services. If we where to use Styra DAS, we don't need to build a highly-available bundle server
+  by ourselves, and can instead build a smaller service to ingest IAM data as data bundles in Styra DAS.
+
+  Another potential we see with Styra DAS is it's ability to interactively work on policies and see how a
+  change affects already executed decisions. It also includes unit testing capabilities which we hope
+  to be able to utilize both interactively and from CI/CD workflows.
```

Comment:
I would suggest that we have a bullet list for what we need to evaluate, just a suggestion tho.